### PR TITLE
REL: Add whatsnew for Zipline 1.2.0

### PR DESCRIPTION
**Pull request by freddiev4**
Just a note:

```
Zipline now requests data for SPY, the default benchmark used for Zipline backtests, 
using the `IEX Trading <https://iextrading.com>__` API, and no longer uses 
``pandas-datareader``. You can run a backtest up to 5 years from the current day 
using this data.
```

That's...sort of true? `pandas-datareader` is still used for this one [test](https://github.com/quantopian/zipline/blob/ca26208569b1465cd86e8fba5c746ca96c32967e/tests/resources/pipeline_inputs/generate.py#L9).

EDIT: That script is for quickly generating Pipeline inputs so I suppose we can just keep it there.

### CODE REVIEW

**Comment by richafrank** [comment]
These updates look great to me!

**Comment by richafrank** [comment]
(I expect they're slightly out of date now.)

**Comment by freddiev4** [comment]
I think the only thing I need to update here is adding the `--bundle-timestamp` fix and latest empyrical bump

**Comment by richafrank** [comment]
Sounds right to me.

**Comment by freddiev4** [comment]
Anddd just now updated the `whatsnew` here

File: docs/source/whatsnew/1.2.0.txt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+Build
+~~~~~
+
+- Update the conda tools we're using to fix our packaging (:issue:`1942`)
+- Upgrade empyrical to 0.3.2 (:issue:`1983`)
+- Update conda tooling and remove Python 3.4 builds (:issue:`2009`)
+- Upgrade empyrical to 0.3.3 (:issue:`2014`)
+- Upgrade empyrical to 0.3.4 (:issue:`2098`)
+- Upgrade empyrical to 0.4.2 (:issue:`2125)
```
Comment (freddiev4) [inline docs/source/whatsnew/1.2.0.txt]: Added this most recently

File: docs/source/whatsnew/1.2.0.txt [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- Add contributing & questions section to the README (:issue:`1889`)
+- Add info about using a conda envs for installs (:issue:`1922`)
+- Fix Beginner Tutorial link (:issue:`1932`)
+- Add clean docs (:issue:`1943`)
+- Add distinct warnings for benchmark and treasury fetchers (:issue:`1971`)
+- Add CONTRIBUTING.rst (:issue:`2033`)
+- Add tutorial on creating a custom ``TradingCalendar`` (:issue:`2035`)
+- Docs & tutorial updates for ingesting, beginners, and csvdir (:issue:`2073`)
+- Documented the new risk and performance metrics API (:issue:`2081`).
+- Fixed a typo in the description of ``--bundle-timestamp`` (:issue:`2123`)
```
Comment (freddiev4) [inline docs/source/whatsnew/1.2.0.txt]: And this

File: docs/source/whatsnew/1.2.0.txt [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+Extensible Risk and Performance Metrics (:issue:`2081`)
+```````````````````````````````````````````````````````
+
+The risk and performance metrics are summarizing values calculated by Zipline
+when running a simulation, for example: returns or Sharpe ratio. 1.1.2
+introduces a new API for registering custom risk and performance metrics defined
+by the user. We have also made it possible to run a backtest without computing
+any metrics to improve the feedback cycle when debugging an algorithm.
+
+For more information, see :ref:`metrics`.
```
Comment (freddiev4) [inline docs/source/whatsnew/1.2.0.txt]: And rebased against master for the above ^^

Comment (richafrank) [reply docs/source/whatsnew/1.2.0.txt]: Nice, what does this link to?

Comment (freddiev4) [reply docs/source/whatsnew/1.2.0.txt]: That links to the `Risk and Performance Metrics` section of the docs that Joe added in his risk metrics PR a while back.

I tested the link locally and it works 🙂

Comment (richafrank) [reply docs/source/whatsnew/1.2.0.txt]: Oh, awesome!

**Comment by richafrank** [comment]
👍 from me

**Comment by freddiev4** [comment]
Thanks @richafrank!

**Pull request merged**